_rules/SC1-2-2-video-has-audio-alternative.md

```diff
@@ -53,7 +56,7 @@ There are no major accessibility support issues known for this rule.
 A video element with an associated track element that contains captions for all the audio.
 
 ```html
-<video src="../test-assets/perspective-video/perspective-video.mp4" controls>
+<video src="../test-assets/perspective-video/perspective-video.mp4.mp4" controls>
```

Comment:
too many `.mp4`? extensions?

Reply:
I would rather change the filename of the test asset

Reply:
Did so. Thanks for the suggestion!